integration/js/pages/AppPage.js

```diff
@@ -493,6 +494,17 @@ class AppPage {
     return expectedState
   }
 
+  async sendDataMessage(message) {
+    const dataMessageSendInput = await this.driver.findElement(elements.dataMessageSendInput);
+    await dataMessageSendInput.clear();
+    await dataMessageSendInput.sendKeys(message);
+    await dataMessageSendInput.sendKeys(Key.ENTER);
+  }
```

Comment:
Should we add a little bit of a wait (say 1 second) to give the message time to be delivered?

Reply:
I added one sec in the test before validating that the message is sent to the other attendee.